com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 - Fixed NetworkList to properly call INetworkSerializable's NetworkSerialize() method (#1682)
 - Fixed The NetworkConfig's checksum hash includes the NetworkTick so that clients with a different tickrate than the server are identified and not allowed to connect (#1728)
 - Fixed OwnedObjects not being properly modified when using ChangeOwnership (#1731)
+- Fixed throwing an exception in OnNetworkUpdate causing other OnNetworkUpdate calls to not be executed. (#1739)
```

Comment:
shouldn't this entry mention OnNetworkSpawn & OnNetworkDespawn instead of OnNetworkUpdate?

Reply:
Yes.